fdbclient/admin_client.go

```diff
@@ -628,7 +628,7 @@ func (client *cliAdminClient) KillProcesses(addresses []fdbv1beta2.ProcessAddres
 	}
 
 	killCommand := fmt.Sprintf(
-		"kill; kill %[1]s; sleep 1; kill %[1]s; sleep 1; kill %[1]s",
+		"kill; kill %[1]s; sleep 1; kill %[1]s; sleep 5",
```

Comment:
where is the `get status request` added in this PR?

Reply:
https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1533#discussion_r1128225854 we discussed this offline. Only the sleep is good enough to give the fdbcli network thread enough time to issue the reboot commands, there is no need for the `status` command.